src/plugins/data_source_management/public/components/data_source_aggregated_view/data_source_aggregated_view.tsx

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import {
+  EuiButtonEmpty,
+  EuiButtonIcon,
+  EuiContextMenu,
+  EuiNotificationBadge,
+  EuiPopover,
+} from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+import { SavedObjectsClientContract, ToastsStart } from 'opensearch-dashboards/public';
+import { DataSourceOption } from '../data_source_selector/data_source_selector';
+import { getDataSourcesWithFields } from '../utils';
+
+interface DataSourceAggregatedViewProps {
+  savedObjectsClient: SavedObjectsClientContract;
+  notifications: ToastsStart;
+  hideLocalCluster: boolean;
+  fullWidth: boolean;
+  activeDatasourceIds?: string[];
+  filterFn?: (dataSource: any) => boolean;
```

Comment:
Would be nice to avoid `any`

Reply:
Thanks, and I change the type to SavedObject type in new commit